core-primitives/substrate-sgx/sp-io/src/lib.rs

```diff
@@ -960,4 +961,29 @@ mod tests {
 	fn storage_set_without_externalities_panics() {
 		storage::set(b"hello", b"world");
 	}
+
+	#[test]
+	fn storage_next_key_works() {
```

Comment:
I like the tests here 👍 Could you add one more case, with an 'empty' externalities?

Reply:
Thanks, I added two test cases, one without ext, and one with empty ext where no `set` is carried out beforehand. I hope that matches what you expect.

Reply:
Nice, yes that's perfect